examples/with-okta-multi-tenant-pkce-flow/api-server/okta.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sync"
+
+	"github.com/supertokens/supertokens-golang/recipe/thirdparty/tpmodels"
+	"github.com/supertokens/supertokens-golang/supertokens"
+
+	verifier "github.com/nirasan/go-oauth-pkce-code-verifier"
+)
+
+type OktaConfig struct {
+	ClientID     string
+	ClientSecret string
+	OktaDomain   string
+}
+
+var oktaVerifiers = sync.Map{}
```

Comment:
Move this into another file and add functions to modify / fetch info from that which this file calls.

Reply:
done